src/static/riot/competitions/editor/_leaderboard.tag

```diff
@@ -83,7 +84,7 @@
                                     <i class="help icon circle"></i>
                                 </span>
                             </label>
-                            <input selenium="key" name="key" value="{_.get(selected_leaderboard, 'key')}" onchange="{ modal_updated }">
+                            <input selenium="key" name="key" value="{_.get(selected_leaderboard, 'key', 'Add')}" onchange="{ modal_updated }">
```

Comment:
`'Add'` as the default key of the selected leaderboard? Was this supposed to be the default submission rule?

Reply:
Right now Null for the submission rule will work the same as Add. It would make sense to set it to the default on the Database as well.